sdk/communication/Azure.Communication.Chat/api/Azure.Communication.Chat.netstandard2.0.cs

```diff
@@ -60,7 +60,7 @@ public partial class ChatMessageReadReceipt
         internal ChatMessageReadReceipt() { }
         public string ChatMessageId { get { throw null; } }
         public System.DateTimeOffset ReadOn { get { throw null; } }
-        public Azure.Communication.CommunicationIdentifier Sender { get { throw null; } }
+        public Azure.Communication.CommunicationIdentifier SenderCommunicationIdentifier { get { throw null; } }
```

Comment:
Let's call it just `Sender`

Reply:
changed

Reply:
We had a discussion with the event grid folks to have names standardized (both on the payload and the name of the properties in the sdk). Will add you both to that discussion, but I can see the benefit of using the suffix CommunicationIdentifier to be super clear that it is an identifier, not a participant or something else. 

Whatever we decide, should be the same on all chat sdks languages.